Source/WebKit/NetworkProcess/cache/NetworkCacheStorage.h

```diff
@@ -129,7 +129,7 @@ class Storage : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Storage,
     size_t approximateSize() const;
 
     // Incrementing this number will delete all existing cache content for everyone. Do you really need to do it?
-    static const unsigned version = 17;
+    static const unsigned version = 18;
```

Comment:
Let's not do this.  We aren't changing anything about cached data.

Reply:
okie.